providers/jenkins/tests/unit/jenkins/operators/test_jenkins_job_trigger.py

```diff
@@ -116,6 +133,10 @@ def test_execute_job_polling_loop(self, parameters, mocker):
 
             operator.execute(None)
             assert jenkins_mock.get_build_info.call_count == 2
+            assert jenkins_mock.get_queue_item.call_count == 4
+
+            assert mock_log.warning.call_count == 2
+            assert mock_log.warning.call_args == mock.call("polling failed, retrying", exc_info=True)
```

Comment:
```suggestion
            assert mock_log.warning.mock_calls == [mock.call("polling failed, retrying", exc_info=True)]
```

it probably looks like this

Reply:
Cool! Got it, will push the changes soon. Thanks for the feedback.